etna/models/nn/deepar_new/loss.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from torch import nn
+from torch.functional import F
+from torch.nn import GaussianNLLLoss
+
+
+class GaussianLoss(GaussianNLLLoss):
+    def __init__(self):
+        super().__init__(reduction="sum")
+
+    def forward(self, inputs, mean, std):
+        return super().forward(input=inputs, target=mean, var=std**2)
+
+    @staticmethod
+    def sample(mean, std):
+        with torch.no_grad():
```

Comment:
You don't need it here, for inference phase it is done by pytorch lightning automatically